src/shared-components/PeopleTabNavbar/PeopleTabNavbar.spec.tsx

```
@@ -0,0 +1,176 @@
+    buttonTestId,
+  }: SearchBarProps) => (
+    <div>
+      <input
+        data-testid={inputTestId ?? 'search-input'}
+        placeholder={placeholder}
+        onChange={(e) => onSearch(e.target.value)}
+      />
+      <button
+        data-testid={buttonTestId ?? 'search-button'}
+        onClick={() => onSearch('clicked')}
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)